tests/conftest.py

```diff
@@ -119,6 +119,12 @@ def pytest_addoption(parser: pytest.Parser):
         "`pip install otx[full]@https://github.com/openvinotoolkit/training_extensions.git@{otx_ref}` will be executed before run, "
         "and reverted after run. Works only for v2.x assuming CLI compatibility.",
     )
+    parser.addoption(
+        "--perf-dir-to-load",
+        type=str,
+        help="Previous performance test directory which contains execution results. "
+        "If training was already done in previous performance test, training is skipped and refer previous result.",
+    )
```

Comment:
How about `resume`, `resume-from` or `resume-dir`? It might be more clear.

Reply:
`resume-from` seems good. I'll update it.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3556/commits/3d6a54e29e75f9e7fc0c36d545d4c25e90881575